spec/requests/api/v1/bracks/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe "Bracks API" do
+  attr_reader :first_closest, :first_furthest, :second_closest, :second_furthest
```

Comment:
i don't think i've ever seen this in tests. what are you gaining by creating attr_reader's for these variables? could you just use the definition of the instance variables in your tests instead?